apps/condo/domains/organization/components/EmployeeRoleSelect.tsx

```diff
@@ -21,13 +20,6 @@ export const EmployeeRoleSelect: React.FC<IEmployeeRoleSelectProps> = (props) =>
         }
     }), [employeeRoles])
 
-    useEffect(()=>{
```

Comment:
is it related to this pr?

Reply:
@pahaz 
Yes! Translation method changes, so I need to fix this logic too